docs/UserGuide.md

```diff
@@ -320,13 +327,32 @@ Furthermore, certain edits can cause the UniLink to behave in unexpected ways (e
+| CONTACT TYPE    | - Should be 'WORK' or 'PERSONAL' <br> - Not case sensitive                                                                                                                                                                                                                                                                                                                                                                                                                           | Work <br> personal                                     |
+| TELEGRAM_HANDLE | - Should begin with '@' <br> - Can contain alphabets, numbers, or underscores <br> - Between 5 and 32 characters (excluding '@')                                                                                                                                                                                                                                                                                                                                                     | @kacey_lim <br> @Maxine123                             |
+| PHONE           | - May start with optional '+' followed by a country code <br> - Should contain only digits <br> - Between 3 and 15 digits long                                                                                                                                                                                                                                                                                                                                                       | 999 <br> +6598765432                                   |
+| EMAIL           | - Must follow the 'local-part@domain' format <br> <br> **Local Part:** <br> - Can include letters, numbers, underscores (_), periods (.), and dashes (-) <br> - Must not start or end with a special character <br> <br> **Domain:** <br> - Must begin with '@', followed by domain labels separated by periods and dashes <br> - Each domain label must start and end with an alphabet or number <br> - Must end with a domain label of at least 2 characters following a period (e.g. .com, .edu) | ryan_teo@example.com <br> caleb-sim@email.edu.sg       |
+| MODULE          | - Should start with 2 to 4 letters, followed by 4 numbers <br> - May end with optional single letter or a dash and one letter                                                                                                                                                                                                                                                                                                                                                        | CS2103-T <br> HSI1000                                  |
```

Comment:
I think currently the module name constraints only allow the module name to start with 2-3 letters and not 4 letters

Reply:
I changed it to 4 letters cause GESS1000 mod (it's under another PR)